app/src/lib/git/push.ts

```diff
@@ -60,10 +69,14 @@ export async function push(
 
   if (!remoteBranch) {
     args.push('--set-upstream')
-  } else if (options !== undefined && options.forceWithLease) {
+  } else if (options.forceWithLease) {
```

Comment:
sorry, this is nit-picky
```suggestion
  } else if (options.forceWithLease === true) {
```